### PR TITLE
create new career page

### PR DESCRIPTION
**Pull request by gceico**

### CODE REVIEW

File: src/pages/careers/web-developer.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+disabled: false
+location: Romania
+date: 2022-04-01T09:09:55.900Z
+permalink: web-developer
+description: Cyscale is a startup with offices in London (UK) and Cluj-Napoca (RO), activating in the cloud cybersecurity industry. We are looking for an experienced Web Developer.
+skills: HTML5, CSS3, JavaScript, GatsbyJS, React.js, Github
+experience: junior/mid
+department: Engineering
+workType: hybrid
+employmentType: part-time
```
Comment (AndreiStefanie) [inline src/pages/careers/web-developer.md:13]: Is it part time?

File: src/pages/careers/web-developer.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+## Job description
+
+You will be part of a dedicated web development team committed to improve users conversion, SEO and product discovery. You will build new website pages, fix/improve existing components, aim for web and SEO good practices, monitor the website performance, track user behavior using various tools.
```
Comment (AndreiStefanie) [inline src/pages/careers/web-developer.md:18]: You will be part of a dedicated web development team committed to improving user conversion, SEO, and product discovery. You will build new website pages, fix/improve existing components, enhance the CMS integration, aim for web and SEO good practices, monitor the website performance, and track user behavior using various tools.

File: src/pages/careers/web-developer.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+-   Provide input in web development area
+-   Ensure the technical feasibility of UI/UX designs
+-   Enhance the performance of the website, optimize for speed and scalability
+
+## Required Skills:
+
+-   Good knowledge and understanding of HTML5, CSS3, Javascript
+-   2+ years of experience as a web developer
+-   Experience with Gatsby, React and TailwindCSS is a plus
+-   Experience with GTM, Google Analytics, Hotjar and Hubspot is a plus
```
Comment (AndreiStefanie) [inline src/pages/careers/web-developer.md:44]: Another item:
- Experience with a headless CMS (we currently use Netlify CMS)

File: src/pages/careers/web-developer.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+title: Web Developer
+disabled: false
+location: Romania
+date: 2022-04-01T09:09:55.900Z
+permalink: web-developer
+description: Cyscale is a startup with offices in London (UK) and Cluj-Napoca (RO), activating in the cloud cybersecurity industry. We are looking for an experienced Web Developer.
+skills: HTML5, CSS3, JavaScript, GatsbyJS, React.js, Github
+experience: junior/mid
+department: Engineering
+workType: hybrid
```
Comment (AndreiStefanie) [inline src/pages/careers/web-developer.md:12]: Why not remote?

File: src/pages/careers/web-developer.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+templateKey: career-page
+title: Web Developer
+disabled: false
+location: Romania
+date: 2022-04-01T09:09:55.900Z
+permalink: web-developer
+description: Cyscale is a startup with offices in London (UK) and Cluj-Napoca (RO), activating in the cloud cybersecurity industry. We are looking for an experienced Web Developer.
+skills: HTML5, CSS3, JavaScript, GatsbyJS, React.js, Github
```
Comment (AndreiStefanie) [inline src/pages/careers/web-developer.md:9]: Also add CMS here, please

File: src/pages/careers/web-developer.md (hunk 1 above)
Comment (ocical) [reply src/pages/careers/web-developer.md:13]: Good catch! No, as it also makes the salary range somehow less relevant.

**Pull request merged**